extensions/mssql/package.json

```diff
@@ -188,6 +188,473 @@
+          "groupName": "Replication",
+          "valueType": "boolean",
+          "defaultValue": null,
+          "objectType": null,
+          "categoryValues": null,
+          "isRequired": false,
+          "isArray": false
+        },
+        {
+          "specialValueType": null,
+          "isIdentity": false,
+          "name": "attachDbFilename",
+          "displayName": "Attach DB filename",
+          "description": null,
+          "groupName": null,
+          "valueType": "string",
+          "defaultValue": null,
+          "objectType": null,
+          "categoryValues": null,
+          "isRequired": false,
+          "isArray": false
+        },
+        {
+          "specialValueType": null,
+          "isIdentity": false,
+          "name": "failoverPartner",
+          "displayName": "Failover partner",
+          "description": "The name or network address of the instance of SQL Server that acts as a failover partner",
+          "groupName": " Source",
+          "valueType": "string",
+          "defaultValue": null,
+          "objectType": null,
+          "categoryValues": null,
+          "isRequired": false,
+          "isArray": false
+        },
+        {
+          "specialValueType": null,
+          "isIdentity": false,
+          "name": "multiSubnetFailover",
+          "displayName": "Multi subnet failover",
+          "description": null,
+          "groupName": null,
+          "valueType": "boolean",
+          "defaultValue": null,
+          "objectType": null,
+          "categoryValues": null,
+          "isRequired": false,
+          "isArray": false
+        },
+        {
+          "specialValueType": null,
+          "isIdentity": false,
+          "name": "multipleActiveResultSets",
+          "displayName": "Multiple active result sets",
+          "description": "When true, multiple result sets can be returned and read from one connection",
+          "groupName": "Advanced",
+          "valueType": "boolean",
+          "defaultValue": null,
+          "objectType": null,
+          "categoryValues": null,
+          "isRequired": false,
+          "isArray": false
+        },
+        {
+          "specialValueType": null,
+          "isIdentity": false,
+          "name": "packetSize",
+          "displayName": "Packet size",
+          "description": "Size in bytes of the network packets used to communicate with an instance of SQL Server",
+          "groupName": "Advanced",
+          "valueType": "number",
+          "defaultValue": null,
+          "objectType": null,
+          "categoryValues": null,
+          "isRequired": false,
+          "isArray": false
+        },
+        {
+          "specialValueType": null,
+          "isIdentity": false,
+          "name": "typeSystemVersion",
+          "displayName": "Type system version",
+          "description": "Indicates which server type system then provider will expose through the DataReader",
```

Comment:
Doesn't the user-facing text here need to be localized? It looks like we don't localize it now, but we could. Is it still possible to localize it if we move it out to this file?

Reply:
I'm not sure what you are asking? Anything in the package.json is localizable. Vscode has a system for doing this. It doesn't look like we are currently localizing anything in the mssql extension, but the account-provider-azure extension has it (its the strings wrapped in %%). We definitely should localize this but since we weren't doing this before I just left it alone.

Reply:
ok, great - just making sure it was still localizable, no need to do it now if we weren't before